CHANGELOG.md

```diff
@@ -1,6 +1,24 @@
 # Changelog
 Changelog for NeoFS Contract
 
+## [0.10.0] - 2021-07-22
```

Comment:
should data be actual(23) or these are all optional little things?

Reply:
It should.